hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/extensions/ITestAbfsDelegationTokens.java

```diff
@@ -264,15 +286,21 @@ public StubAbfsTokenIdentifier verifyCredentialsContainsToken(
     Token<? extends TokenIdentifier> token = credentials.getToken(
         new Text(serviceName));
 
-    assertEquals("Token Kind in " + token,
-        StubAbfsTokenIdentifier.TOKEN_KIND, token.getKind());
-    assertEquals("Token Service Kind in " + token,
-        tokenService, token.getService().toString());
+    Assertions.assertThat(token)
+        .describedAs("No token found for %s", serviceName)
+        .isNotNull();
+
+    Assertions.assertThat(token.getKind())
+        .describedAs("Token Kind in %s", token)
+        .isEqualTo(StubAbfsTokenIdentifier.TOKEN_KIND);
+    Assertions.assertThat(token.getService().toString())
```

Comment:
assert for isNotNull for token.getService() before toString()?

Reply:
good suggestion